views/add-event.hbs

```diff
@@ -0,0 +1,66 @@
+<div class="add-eventContainer">
+  <h3 class="userAddEvent">{user name} add event</h3>
+
+  <form class="add-event-form" action="/add-event" method="post">
+    <label>Title: </label>
+    <input class="form-control" type="text" name="name" placeholder="title" required>
+
+    <label>Picture: </label>
+    <input class="form-control" type="text" name="imageUrl" placeholder="choose picture">
+
+    <label>Start Date:</label>
+    <input class="form-control" type="date" name="startDate" placeholder="example year-month-day" required>
+
+    <label>End Date:</label>
+    <input class="form-control" type="date" name="endDate" placeholder="example year-month-day" required>
+
+    <label>Start Time: </label>
+    <input class="form-control" type="text" name="startTime" placeholder="example 00:00" required>
+
+    <label>End Time: </label>
+    <input class="form-control" type="text" name="endTime" placeholder="example 00:00" required>
+
+    <label>Place: </label>
+
+    <label class="sel1">Category: </label>
+    <select class="options form-control" name="categories" required>
+      <option disabled selected>Select option..</option>
+      <option value="music">Music</option>
+      <option value="dining">Dining</option>
+      <option value="educational">Educational</option>
+      <option value="conference">Conference</option>
+      <option value="sport">Sport</option>
+      <option value="competition">Competition</option>
+      <option value="launch">Launch</option>
+      <option value="party">Party</option>
+      <option value="wedding">Wedding</option>
+      <option value="cultural event">Cultural event</option>
+      <option value="Miscellaneous">Miscellaneous</option>
+    </select>
+
+    <label>Accessibility: </label>
+    <select class="accessibility form-control" name="accessibilityOptions" required>
```

Comment:
this should be a multi-select...

Reply:
so should categories. see https://github.com/foundersandcoders/open-tourism-platform/blob/master/docs.md#create-event